src/modules/aws_ecr/index.ts

```diff
@@ -2,7 +2,7 @@ import {
   ECR,
   Repository as RepositoryAws,
   paginateDescribeRepositories,
-  Image as ImageAws,
+  ImageDetail,
```

Comment:
is this a superset if `Image`?

Reply:
not a straight super set, there are some minor differences between the two